src/app/ubs-admin/components/ubs-admin-order-details-form/ubs-admin-order-details-form.component.html

```diff
@@ -78,45 +78,45 @@ <h3 class="form__headline pl-3">Деталі замовлення</h3>
           <span class="col-sm-2"></span>
         </li>
         <li class="d-flex justify-content-between align-items-baseline py-3">
-          <span class="col-sm-2">Сума до оплати</span>
-          <span class="col-sm-2 offset-sm-4 text-center">300 грн</span>
-          <span class="col-sm-2 text-center">550 грн</span>
-          <span class="col-sm-2 text-center" *ngIf="isVisible">400 грн</span>
+          <span class="col-sm-2">{{ "order-details.sum-to-pay" | translate }}</span>
+          <span class="col-sm-2 offset-sm-4 text-center">300 {{ "" | localizedCurrency }}</span>
+          <span class="col-sm-2 text-center">550 {{ "" | localizedCurrency }}</span>
+          <span class="col-sm-2 text-center" *ngIf="isVisible">400 {{ "" | localizedCurrency }}</span>
           <span class="col-sm-2"></span>
         </li>
         <li class="d-flex justify-content-end align-items-baseline py-2">
-          <span class="col-sm-3 text-danger pl-5" *ngIf="payMore">Потрібно доплатити 250 грн</span>
+          <span class="col-sm-3 text-danger pl-5" *ngIf="payMore">{{ "order-edit.need-to-pay" | translate }} 250 {{ "" | localizedCurrency }}</span>
           <span class="col-sm-4 text-danger pl-5" *ngIf="payMore">
-            Замовлення не може бути змінене. Мінімальна сума замовлення 500 грн
+            {{ "order-details.min-sum" | translate }} 500 {{ "" | localizedCurrency }}
           </span>
         </li>
       </ul>
     </div>
     <div class="certificate">
       <div class="form__block col-lg-3 col-md-6 col-sm-6">
         <div class="form-group">
-          <label class="form__label py-2" for="cert-num">Номер сертифікату</label>
+          <label class="form__label py-2" for="cert-num">{{ "order-details.certificate-number-label" | translate }}</label>
           <input class="form-control" type="text" id="cert-num" placeholder="XXXX-XXXX" formControlName="certificate" />
         </div>
         <div class="form-group">
-          <label class="form__label py-2" for="order-num">Номер замовлення з магазину</label>
+          <label class="form__label py-2" for="order-num">{{ "order-details.order-number-label" | translate }}</label>
           <input class="form-control" type="text" id="order-num" placeholder="XXXX-XXXX" formControlName="storeOrderNumber" />
         </div>
       </div>
     </div>
     <div class="bottom d-flex align-items-baseline py-3">
       <div class="form-floating col-sm-6">
-        <label for="textarea">Коментар до замовлення від клієнта</label>
+        <label for="textarea">{{ "order-details.client-comment.label" | translate }}</label>
         <textarea
           class="form-control comment"
           formControlName="customerComment"
-          placeholder="Будь-які деталі, які Ви хотіли б повідомити додатково щодо цього замовлення"
+          placeholder="{{ 'order-details.client-comment.placeholder' | translate }}"
           id="textarea"
```

Comment:
Let's rename 'id' and 'for'  to customer-comment

Reply:
Done